COBOL Programming Course #2 - Advanced Topics/COBOL Programming Course #2 - Advanced Topics.md

```diff
@@ -225,6 +225,115 @@ COMP-2 supersedes COMP-1 for more precise scientific data storage as well as com
 
 **Note** : [This](https://www.ibm.com/support/pages/how-display-hexadecimal-using-cobol) COBOL program can display the hexadecimal contents (and hence the exact internal representation) of a field. You can declare binary, packed decimal or zoned variable (or anything else, for that matter), do arithmetic with them and use the program to see how they are internally stored. 
 
+\newpage
+# Dynamic-Length Item
+So far during this course, we have only explored data items that have a fixed length. In other words, you need to define the length you need for each data type. But in this section, we will explore a fairly new concept in Enterprise COBOL - dynamic-length items.
+
+Enterprise COBOL v6.3 supports dynamic-length items, which are items whose logical length might change at runtime.
+
+## Dynamic-Length Elementary Items
```

Comment:
We should define the term elementary in this section. It is an item that cannot be further subdivided. These items have a PIC clause because they reserve the storage for the item.

Reply:
I believe the term elementary was also used on Course 1. But again, no definition there. Are Course 2 being constructed under the assumption that those who read it have previously done Course 1? If so, will try to fit the definition somewhere on Course 1.

Reply:
Originally I was thinking that it should be defined in this course as well. However, I agree with your point that readers should already have this knowledge or refer to the introductory course. Therefore, I think we should just include the definition in the `Data Division` chapter of the introductory course. That can be a separate PR if you want.